nerfums-app/src/app/auth/auth.module.ts

```diff
@@ -0,0 +1,12 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { UserLoginComponent } from './user-login/user-login.component';
+import { RegisterUserComponent } from './register-user/register-user.component';
+@NgModule({
```

Comment:
Nice, this is good because any auth related stuff can go in a separate module, which could potentially be packaged and distributed to be shared amongst multiple apps.